modules/digiTrustIdSystem.js

```diff
@@ -0,0 +1,307 @@
+/**
+ * This module adds DigiTrust ID support to the User ID module
+ * The {@link module:modules/userId} module is required
+ * If the full DigiTrust Id library is included the standard functions
+ * will be invoked to obtain the user's DigiTrust Id.
+ * When the full library is not included this will fall back to the
+ * DigiTrust Identity API and generate a mock DigiTrust object.
+ * @module modules/digiTrustIdSystem
+ * @requires module:modules/userId
+ */
+
+// import { config } from 'src/config';
+import * as utils from '../src/utils'
+import { ajax } from 'src/ajax';
+import { attachIdSystem } from '../modules/userId';
+// import { getGlobal } from 'src/prebidGlobal';
+
+/**
+ * Checks to see if the DigiTrust framework is initialized.
+ * @function
+ */
+function isInitialized() {
+  if (window.DigiTrust == null) {
+    return false;
+  }
+  return DigiTrust.isClient; // this is set to true after init
+}
+
+/**
+ * Tests for presence of the DigiTrust object
+ * */
+function isPresent() {
+  return (window.DigiTrust != null);
+}
+
+var noop = function () {
+};
+
+const MAX_RETRIES = 4;
```

Comment:
suggest changing this value to 1 :)

Reply:
We could go to two. There are two code paths that could possibly run prior to DigiTrust being loaded. It may be an unnecessary guard, but only boils down to 300 milliseconds.

Reply:
Changed to "2"